testing/taxonomies/MessageShape.ttl

```diff
@@ -46,8 +46,10 @@ shapes:ResponseMessageShape
 		sh:nodeKind sh:IRI ;
         #sh:class ids:Message ;
 		sh:pattern "^(?:(http(s)?|idscp|(s)?ftp))";
-		sh:minCount 1 ;
-		sh:maxCount 1 ;
+		# --- @PNL: ---
+		# sh:minCount 1 ;
```

Comment:
This doesn't look right to me. ResponseMessages should have exactly one correlation message. Other messages may have a correlationMessage, but not strictly required

Reply:
I agree. This is just from last week's testing and not cleaned up in this version yet. I already corrected that part locally and it will be back in in the next commit.